README.md

```diff
@@ -109,8 +109,30 @@ Transport
 
 py_zipkin (for the moment) thrift-encodes spans. The actual transport layer is
 pluggable, though. The `transport_handler` is a function that takes a single
-argument - the thrift-encoded bytes. An example of sending spans to Kafka
-using the [kafka-python](https://pypi.python.org/pypi/kafka-python) package:
+argument - the thrift-encoded bytes.
+
+The simplest way to get spans to the collector is via HTTP POST. Here's an
+example of a simple HTTP transport using the `requests` library. This assumes
+your Zipkin collector is running at localhost:9411.
+
+```python
+import requests
+
+def http_transport(message):
```

Comment:
![screenshot from 2016-10-12 08 15 10](https://cloud.githubusercontent.com/assets/11739152/19294103/09335fc4-9054-11e6-9ab0-73668cc5f801.png)

Hi, I checked. but it raise new error: py_zipkin.exception.ZipkinError: Root spans require a transport handler to be given. 
Please check, thanks so much :D